addons/mrp_subcontracting_dropshipping/models/res_company.py

```diff
@@ -40,7 +77,32 @@ def create_missing_subcontracting_dropshipping_rules(self):
         company_todo_rules = company_ids - company_has_rules
         company_todo_rules._create_subcontracting_dropshipping_rules()
 
+    @api.model
+    def create_missing_subcontracting_dropshipping_sequence(self):
+        company_ids = self.env['res.company'].search([])
+        company_has_seq = self.env['ir.sequence'].search([('code', '=', 'stock.subcontracting.dropshipping')]).mapped('company_id')
+        company_todo_sequence = company_ids - company_has_seq
+        company_todo_sequence._create_subcontracting_dropshipping_sequence()
+
+    @api.model
+    def create_missing_subcontracting_dropshipping_picking_type(self):
+        company_ids = self.env['res.company'].search([])
+        company_has_dropship_subcontractor_picking_type = self.env['stock.picking.type'].search([
+            ('default_location_src_id.usage', '=', 'supplier'),
+            ('default_location_dest_id', 'in', company_ids.subcontracting_location_id.ids),
+        ]).mapped('company_id')
+        company_todo_picking_type = company_ids - company_has_dropship_subcontractor_picking_type
+        company_todo_picking_type._create_subcontracting_dropshipping_picking_type()
+
+    def _create_per_company_sequences(self):
+        super()._create_per_company_sequences()
+        self.create_missing_subcontracting_dropshipping_sequence()
```

Comment:
Should be `_create_subcontracting_dropshipping_sequence`, since this is called on newly created company (i.e. we don't want to create it for existing companies)